### PR TITLE
Add alertmanager2gelf Alertmanager webhook

### PR DESCRIPTION
**Pull request by c35sys**
alertmanager2gelf - Prometheus Alertmanager webhook to gelf

### CODE REVIEW

File: content/docs/operating/integrations.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ data volumes.
 For notification mechanisms not natively supported by the Alertmanager, the
 [webhook receiver](/docs/alerting/configuration/#webhook_config) allows for integration.
 
+  * [alertmanager2gelf](https://github.com/b-com-software-basis/alertmanager2gelf): Prometheus Alertmanager webhook to gelf
```
Comment (brian-brazil) [inline content/docs/operating/integrations.md]: This should be listed as GELF

Comment (c35sys) [reply content/docs/operating/integrations.md]: Done

**Comment by brian-brazil** [comment]
Can you add the DCO?

**Comment by c35sys** [comment]
DCO ok

File: content/docs/operating/integrations.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Alertsnitch](https://gitlab.com/yakshaving.art/alertsnitch): saves alerts to a MySQL database
   * [AWS SNS](https://github.com/DataReply/alertmanager-sns-forwarder)
   * [DingTalk](https://github.com/timonwong/prometheus-webhook-dingtalk)
+  * [GELF](https://github.com/b-com-software-basis/alertmanager2gelf): Prometheus Alertmanager webhook to gelf
```
Comment (brian-brazil) [inline content/docs/operating/integrations.md]: You don't need the note here, these are all webhooks to something so it's implied that this is what this is.

Comment (c35sys) [reply content/docs/operating/integrations.md]: I removed the note.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**